<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Excel</class>
 <widget class="QWidget" name="Excel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>511</width>
    <height>440</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTextBrowser" name="info">
   <property name="geometry">
    <rect>
     <x>38</x>
     <y>170</y>
     <width>431</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
  <widget class="QSplitter" name="splitter">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>390</y>
     <width>186</width>
     <height>28</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <widget class="QPushButton" name="checkFile">
    <property name="text">
     <string>检查</string>
    </property>
   </widget>
   <widget class="QPushButton" name="convert">
    <property name="text">
     <string>开始转换</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>22</x>
     <y>21</y>
     <width>471</width>
     <height>57</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="inputLabel">
      <property name="text">
       <string>输入文件</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="inputLineEdit"/>
    </item>
    <item row="0" column="2">
     <widget class="QToolButton" name="inputSelect">
      <property name="text">
       <string>...</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="outputLabel">
      <property name="text">
       <string>输出文件夹</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="outputLineEdit"/>
    </item>
    <item row="1" column="2">
     <widget class="QToolButton" name="outputSelect">
      <property name="text">
       <string>...</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>471</width>
     <height>26</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QCheckBox" name="sameStandard">
      <property name="text">
       <string>统一标准</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="processStandardLabel">
      <property name="text">
       <string>处理标准</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QLineEdit" name="processStandard"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>471</width>
     <height>26</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0">
     <widget class="QCheckBox" name="header">
      <property name="text">
       <string>有表头</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QCheckBox" name="sheet">
      <property name="text">
       <string>多sheet</string>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QLineEdit" name="processNumberInput"/>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="headerNum"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
